internal/test/e2e/e2e_test.go

```diff
@@ -683,8 +683,9 @@ func scaleDownTests(t *testing.T, kubectl *kubectlContext) {
 	require.NoError(t, err, out)
 
 	t.Log("If the cluster is scaled down")
-	const expectedReplicas = 1
-	err = kubectl.Scale("etcdcluster/my-cluster", expectedReplicas)
+	const expectedReplicasNew = 1
+	require.True(t, expectedReplicasNew < expectedReplicasOriginal, "Must use a lower replicas count")
```

Comment:
Is this check necessary? Both `expectedReplicasNew` and `expectedReplicasOriginal` are consts, so cannot change, which means this always evaluates to true.

Reply:
It's there to catch the case where someone in future changes one of these values without also changing the other.